lib/natalie/compiler2/pass1.rb

```diff
@@ -78,6 +78,66 @@ def transform_call(exp, used:, with_block: false)
         instructions
       end
 
+      def transform_case(exp, used:)
+        _, var, *whens, else_block = exp
+        return transform_expression(else_block, used: used) if whens.empty?
+        instructions = []
+        if var
+          instructions << transform_expression(var, used: true)
+          whens.each do |when_statement|
+            # case a
+            # when b, c, d
+            # =>
+            # if (a === b || a === c || a === d)
+            _, options, body = when_statement
+
+            options[1...].each do |option|
```

Comment:
I dislike using the `[1...]` syntax for this, as it makes it hard to read. Why skip the first one? (Don't answer that in a comment please -- make it clear in the code.)

In this case, should we destructure the `options` array so we can label the parts?